### PR TITLE
2264 update win10 installation instructions in README.md

### LINKED ISSUE #2264: README.md: update Windows10 install instructions
The windows 10 install instructions are out-of-date and should be updated.

### PR DESCRIPTION
**Pull request by hytung8**
Closes #2264

Checklist:
 I have read Giveth's contribution guidelines.
 My pull request has a descriptive title.
 I have viewed these changes either locally on my machine or on github.

Details:
README.md:  I updated the Windows10 installation procedure based on steps suggested by Ramin.  I added links to Discord (in addition to Element) and that Node v10.24.1 should be used (removing references to Node v8).  I also suggested installing git (for those that installed github desktop).  I uploaded a screenshot VSstudio.png which is referenced in the README.md Windows10 install procedure.

### CODE REVIEW

**Comment by aminlatifi** [comment]
@hytung8 Thanks a lot.

File: README.md [hunk 1]
```diff
@@ -128,7 +119,7 @@ REACT_APP_DECIMALS | 8 | How many decimal should be shown for cryptocurrency val
 REACT_APP_FEATHERJS_CONNECTION_URL | Differs per REACT_APP_ENVIRONMENT | Overwrites the environment injected feathers connection URL. |
 REACT_APP_NODE_CONNECTION_URL | Differs per REACT_APP_ENVIRONMENT | Overwrites the EVM node connection URL for making EVM transactions. |
 REACT_APP_LIQUIDPLEDGING_ADDRESS | Differs per REACT_APP_ENVIRONMENT | Overwrites the Liquid Pledging contract address. |
-REACT_APP_DAC_FACTORY_ADDRESS | Differs per REACT_APP_ENVIRONMENT | Overwrites the Communities contract address. |
+REACT_APP_DAC_FACTORY_ADDRESS | Differs per REACT_APP_ENVIRONMENT | Overwrites the DACs contract address. |
```
Comment (RamRamez) [inline README.md:122]: We've changed the "DACs" word to "Communities" and "Milestones" word to "Traces".
Please update your version with the latest README file.

File: README.md [hunk 2]
```diff
@@ -159,8 +150,8 @@ The following events are tracked with Google Analytics:
 - pageviews
 - signup 
 - signin
-- creation of Communities/Campaigns/Traces
-- editing of Communities/Campaigns/Traces
+- creation of DACs/Campaigns/Milestones
+- editing of DACs/Campaigns/Milestones
```
Comment (RamRamez) [inline README.md:154]: Also here

**Review by RamRamez — CHANGES_REQUESTED** [review]
Thanks @hytung8

**Comment by RamRamez** [comment]
It's done in PR #2297

**Pull request closed without merging**